app/api/models.py

```diff
@@ -0,0 +1,84 @@
+from app import db
+
+class User(db.Model):
+    __tablename__ = "users"
+    id = db.Column(db.Integer,primary_key=True)
+    name = db.Column(db.String(64),nullable=False)
+    email = db.Column(db.String(64),unique=True, index=True, nullable=False)
+    password = db.Column(db.String(128), nullable=False)
+    role_id = db.Column(db.Integer, db.ForeignKey('roles.id'))

```

Comment:
```suggestion
    role_id = db.Column(db.Integer, db.ForeignKey('roles.id'), nullable=False)

```

So that we dont allow storing of users without role_id

Reply:
shouldn't it be nullable=True

Reply:
oops,was abit confused,my bad